cloud-config/main.yml

```diff
@@ -43,60 +43,7 @@
         security_groups:
         - ((terraform_outputs.bosh_security_group))
 
-- type: replace
-  path: /vm_types/-
-  value:
-    name: admin-ui
-    cloud_properties:
-      instance_type: t2.medium
-      ephemeral_disk:
-        size: 10240
-- type: replace
-  path: /vm_types/-
-  value:
-    name: shibboleth
-    cloud_properties:
-      instance_type: m3.medium
-      ephemeral_disk:
-        size: 10240
-- type: replace
-  path: /vm_types/-
-  value:
-    name: logsearch_es_master
-    cloud_properties:
-      instance_type: t2.xlarge
-      ephemeral_disk:
-        size: 15000
-- type: replace
-  path: /vm_types/-
-  value:
-    name: logsearch_es_data
-    cloud_properties:
-      instance_type: r4.xlarge
-- type: replace
-  path: /vm_types/-
-  value:
-    name: logsearch_redis
-    cloud_properties:
-      instance_type: t2.medium
-- type: replace
-  path: /vm_types/-
-  value:
-    name: logsearch_ingestor
-    cloud_properties:
-      instance_type: r4.large
-- type: replace
-  path: /vm_types/-
-  value:
-    name: logsearch_kibana
-    cloud_properties:
-      instance_type: t2.large
-- type: replace
-  path: /vm_types/-
-  value:
-    name: logsearch_maintenance
-    cloud_properties:
-      instance_type: t2.large
+# todo (mxplusb): get rid of these once k8s is using aws type aliases.
```

Comment:
The diff isn't rendering properly, I would recommend looking at this natively if there are questions or concerns.